float64 pole_timestamp
float64[2] pole_angle
float64[2] pole_dangle

float64 franka_timestamp
float64[3] franka_effector_position
float64[3] franka_effector_velocity

float64 control_timestamp
float64[3] control_effector_position
float64[3] control_effector_velocity
float64[3] control_effector_acceleration

bool reset